packages/menu/src/MenuItem.tsx

```diff
@@ -172,11 +172,11 @@ interface ButtonMenuItemProps
   href?: null;
 }
 
-type MenuItemProps = LinkMenuItemProps | ButtonMenuItemProps;
+type MenuItemProps = (LinkMenuItemProps | ButtonMenuItemProps) & {
+  [key: string]: any;
```

Comment:
What prop are we passing here that's not being caught by Link/ButtonMenuItemProps? Open to handling as above if necessary, but curious about what's causing the error

Reply:
Good catch, this was from when I was debugging something and completely forgot about this. Sorry!

Reply:
👍